src/pages/gateway/create-mesh.md

```diff
@@ -76,6 +76,18 @@ Workspaces that already have an API mesh will have "API Mesh" displayed on their
 
 Refer to the [command reference] for a detailed description of `aio api-mesh:create`.
 
+### Access the gateway
+
+The `aio api-mesh:create` response automatically assigns you an API key and subscribes that API key to the mesh service. You can also retrieve the API key by viewing the project in the [Adobe Developer Console].
+
+After you [create a mesh], you can access the GraphQL endpoint in any GraphQL browser by modifying the following URL: `https://graph.adobe.io/api/<meshId>/graphql?api_key=<your_apiKey>`
+
+<InlineAlert variant="info" slots="text"/>
+
+For security purposes, we recommend moving your API key from the provided link into a request header.
+
+The `aio api-mesh:create` response provides the exact url to access the gateway for your mesh.
```

Comment:
```suggestion
The `aio api-mesh:create` response provides the exact URL to access the gateway for your mesh.
```

Is there any difference between lines 83 and 89? If no, I recommend consolidating the sentences. If yes, then maybe you need a bulleted list that describes the response of the `create` command.